src/dbnode/persist/fs/volume.go

```diff
@@ -0,0 +1,51 @@
+package fs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/m3db/m3/src/x/ident"
+)
+
+// GetVolumeIndices returns available volume indices for specified options.
+func GetVolumeIndices(
```

Comment:
nit: "VolumeIndices" perhaps? Since Go is so [allergic to getters](https://golang.org/doc/effective_go.html#Getters)? (hah)

Reply:
Ah, this is actually some unused code. I'm going to remove it.